src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -180,78 +180,78 @@ function LibraryServiceApi(gaxGrpc, grpcClients, opts) {
 // Path templates
 
 var SHELF_PATH_TEMPLATE = new gax.PathTemplate(
-    'shelves/{shelf}');
+    'shelves/{shelf_id}');
 
 var BOOK_PATH_TEMPLATE = new gax.PathTemplate(
-    'shelves/{shelf}/books/{book}');
+    'shelves/{shelf_id}/books/{book_id}');
 
 var ARCHIVED_BOOK_PATH_TEMPLATE = new gax.PathTemplate(
-    'archives/{archive_path=**}/books/{book}');
+    'archives/{archive_path=**}/books/{book_id}');
 
 /**
  * Returns a fully-qualified shelf resource name string.
- * @param {String} shelf
+ * @param {String} shelf_id
  * @returns {String}
  */
-LibraryServiceApi.prototype.shelfPath = function(shelf) {
+LibraryServiceApi.prototype.shelfPath = function(shelfId) {
   return SHELF_PATH_TEMPLATE.render({
-    shelf: shelf
+    shelf_id: shelfId
```

Comment:
It looks like there is a casing bug in node js... the doc doesn't match the real parameter.